client/layout/activity-panel/panels/inbox.js

```diff
@@ -4,8 +4,9 @@
  */
 import { __ } from '@wordpress/i18n';
 import { Button } from '@wordpress/components';
-import { Component, Fragment } from '@wordpress/element';
+import { Component, Fragment, compose } from '@wordpress/element';
```

Comment:
I'm getting a warning in the console, although this problem is surely in several places

>wp.element.compose is deprecated and will be removed in 3.5. Please use wp.compose.compose instead.